gdx/src/com/badlogic/gdx/graphics/g2d/PixmapPacker.java

```diff
@@ -342,9 +346,18 @@ public synchronized TextureAtlas generateTextureAtlas (TextureFilter minFilter,
 	/** Updates the {@link TextureAtlas}, adding any new {@link Pixmap} instances packed since the last call to this method. This
 	 * can be used to insert Pixmap instances on a separate thread via {@link #pack(String, Pixmap)} and update the TextureAtlas on
 	 * the rendering thread. This method must be called on the rendering thread. After calling this method, disposing the packer
-	 * will no longer dispose the page pixmaps. */
+	 * will no longer dispose the page pixmaps. Has useIndexes on by default so as to keep backwards compatibility*/
```

Comment:
Why is the default `true`?
How does this preserve backwards compatibility? 
The code that uses useIndex has been newly added.
So shouldn't it be off?

Reply:
Because it's imitating the TexturePacker tool, and the tool has it on by default.
Also, the only reason it would break would be if you have been naming your assets with _01.png, _02.png, which could be argued to be a mistake in the first place, because this is the convention used by TexturePacker and PixmapPackerIO in order to create animations.
Ultimately, although it IS a new way of the class working, it's more bringing it inline with how all other texture packing in the libgdx world works, so I see it more as a bug fix, than a new feature.
But I can understand your view. If others agree, it can be swapped out to being false.
My thought process would be, it's better to fix the bug, than to keep it in to preserve backwards compatibility. Though that's only my own personal thoughts.